package.json

```diff
@@ -1,28 +1,42 @@
 {
   "name": "linter-rubocop",
-  "main": "./lib/index.coffee",
-  "version": "0.5.3",
+  "main": "./src/index.js",
+  "version": "0.6.0",
   "description": "Lint Ruby on the fly, using rubocop",
   "repository": "https://github.com/AtomLinter/linter-rubocop",
   "license": "MIT",
   "engines": {
-    "atom": ">=1.0.0 <2.0.0"
+    "atom": ">=1.4.0 <2.0.0"
+  },
+  "configSchema": {
+    "command": {
+      "type": "string",
+      "default": "rubocop",
+      "description": "\n This is the absolute path to your `rubocop` command. You may need to run \n `which rubocop` or `rbenv which rubocop` to find this. Examples: \n `/usr/local/bin/rubocop` or `/usr/local/bin/bundle exec rubocop --config \n /my/rubocop.yml`. \n"
```

Comment:
Are these newlines really necessary? Atom should be handling the display of this for us.